src/utils/files.ts

```diff
@@ -83,7 +83,8 @@ export async function withTempDir<T>(
     return await callback(directory);
   } finally {
     if (cleanup) {
-      await rimraf(directory);
+      // Intentionally DO NOT await removal
```

Comment:
Also here: why the change?

Reply:
Similar to the one above: when I did a dry run, finalizing publish failed on the removal of a temp directory. This is to avoid that.